### PR TITLE
docs: format all config demos [skip ci]

### PR DESCRIPTION
**Pull request by chenjiahan**
# PR Details

## Description

- Format all config demos, using `export default defineConfig(` by default.
- It's verbose to declare `import { defineConfig } from '@modern-js/app-tools'` in every demo, so I removed this line.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: website/docs/apis/config/bff/prefix.md [hunk 1]
```diff
@@ -28,9 +28,7 @@ api
 
 该配置选项可以修改默认的路由前缀：
 
-```javascript title="modern.config.js"
-import { defineConfig } from '@modern-js/app-tools';
-
+```js title="modern.config.js"
```
Comment (caohuilin) [inline website/docs/apis/config/bff/prefix.md:31]: 这里文件名都改成 modern.config.ts 吧，在 js 文件中使用 export default 会报错

Comment (chenjiahan) [reply website/docs/apis/config/bff/prefix.md:31]: 了解，我改一下。

不过理想情况下 js 文件里使用 export default 也应该是允许的（都走一遍转义），vite 就是两者都统一使用 export default。

Comment (caohuilin) [reply website/docs/apis/config/bff/prefix.md:31]: 嗯，后面优化一下吧

Comment (chenjiahan) [reply website/docs/apis/config/bff/prefix.md:31]: 我看了一下 loadConfig 的代码实现，现在 .js 文件也支持 export default 滴，会统一走 bundleRequire 的逻辑.

我在工程里试了下也正常，那我在文档就统一写 export default 了。

Comment (chenjiahan) [reply website/docs/apis/config/bff/prefix.md:31]: 等 module-tool 也支持 modern.config.ts 后，我再把文件名都改成 modern.config.ts

**Review by caohuilin — APPROVED** [review]

**Pull request merged**